### DISCUSSION TITLE
Sticky footer

### DISCUSSION THREAD

**Discussion by marinsokol5** [facebook/docusaurus#9230] [General]
Hey, 

Is it possible to have the footer sticky (stays on the screen), same as with header, without swizzling the Layout component? 
I saw https://github.com/facebook/docusaurus/issues/7746 so it seems it previously worked like this but it no longer does?

**Comment by marinsokol5** [comment]
I guess the answer is that we should do that through custom styling, like
```
.footer {
    position: sticky;
    bottom: 0;
    z-index: var(--ifm-z-index-fixed);
}
```

which is then preffered over swizzling, right?

**Reply by slorber** [reply]
You can use CSS or swizzle, it's up to you, but keep in mind that both can break unless we mark the component stable to swizzle or the CSS classname as stable (see https://docusaurus.io/docs/styling-layout#theme-class-names)

We'll mark more and more things as stable in the future and ensure major version retro-compatibility
